Amplify/DevMenu/Data/LogEntryItem.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Data class for each log item in Log Viewer Screen
+struct LogEntryItem: Identifiable, Hashable {
```

Comment:
should this have an optional `error` or `stackTrace` as well? @royjit thoughts?

Reply:
I dont think we can get the `stackTrace`. We can always add this later, I think we are good now, unless we know what to do with the `error`.